### PR TITLE
Ankita Added Cycle_Detection_In_Linked_List.java

### PR DESCRIPTION
**Pull request by ankitasantape**
Implemented the code of Cycle Detection in Linked List using Java
#189
[x] - Code is well indented.
[x] - Code is well commented.
[x] - Code is not plagiarized.
[x] - Code is placed in the proper folder.
#GSSOC'21

### CODE REVIEW

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
link the code to the folder, link the issue and also check your code once. Complexity analysis has to come at the bottom.

**Comment by ankitasantape** [comment]
I have made the requested changes, please review it.

File: Code/Java/Cycle_Detection_In_Linked_List.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        }
+    }
+    static boolean hasCycle(SinglyLinkedListNode head) {
+            if(head == null ){
+                 return false;
+              }
+              SinglyLinkedListNode slow = head;
+              SinglyLinkedListNode fast = head.next;
+              while(slow != fast) {
+                    if(fast == null || fast.next == null) {
```
Comment (tarun26091999) [inline Code/Java/Cycle_Detection_In_Linked_List.java:48]: check this condition once.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
link your PR with the issue.

File: Code/Java/Cycle_Detection_In_Linked_List.java (hunk 1 above)
Comment (ankitasantape) [reply Code/Java/Cycle_Detection_In_Linked_List.java]: I have updated the condition, please review it.

**Comment by tarun26091999** [comment]
> link your PR with the issue.

@ankitasantape this is not done yet.

**Review by plazzy99 — APPROVED** [review]
code LGTM 👍🏻

**Review by tarun26091999 — APPROVED** [review]

**Pull request merged**